pkg/cmd/util/tokencmd/negotiator_gssapi.go

```diff
@@ -155,12 +156,39 @@ func (g *gssapiNegotiator) Release() error {
 	return utilerrors.NewAggregate(errs)
 }
 
+var libPaths = map[string][]string{
+	"darwin": []string{
+		"libgssapi_krb5.dylib",
+	},
+	"linux": []string{
+		// MIT
+		"libgssapi_krb5.so.2.2",
+		"libgssapi_krb5.so.2",
+		"libgssapi_krb5.so",
```

Comment:
Are we relying on any feature that's not available from libgssapi_krb5.so.2.0? If not, we should probably not explicitly try to load `libgssapi_krb5.so.2.2`.

Similarly, we should *never* try to load the unversioned shared object, because if it jumps to `.so.3` at some point, that means that backwards-compatibility has been broken and we shouldn't assume that it works.